common/v2/components/WalletUnlock/Mnemonic.tsx

```diff
@@ -146,7 +146,7 @@ class MnemonicDecryptClass extends PureComponent<OwnProps, State> {
       return;
     }
 
-    const seed = await mnemonicToSeed(formattedPhrase, pass).toString('hex');
+    const seed = await mnemonicToSeedSync(formattedPhrase, pass).toString('hex');
```

Comment:
What's the reason for using `mnemonicToSeedSync` here?

Reply:
`mnemonicToSeed` is now Async
so to keep the same behaviour I used `mnemonicToSeedSync`
https://github.com/bitcoinjs/bip39/blob/master/CHANGELOG.md